spring-context/src/main/java/org/springframework/context/aot/ReflectiveProcessorAotContributionBuilder.java

```diff
@@ -20,8 +20,8 @@
 import java.util.HashSet;
 import java.util.LinkedHashSet;
 import java.util.Set;
-import java.util.stream.StreamSupport;
 
+import com.google.common.collect.Streams;
```

Comment:
Apart from the `commons-logging` and `micrometer-api` dependencies there are no 3rd party dependencies in Spring. I doubt adding a dependency on Google Guava is going to happen anytime soon.